BP/common/culture/cultures/00_central_semitic.txt

```diff
@@ -0,0 +1,1140 @@
+			{ "dynnp_al-" "dynn_Baghdadi" }
+			{ "dynnp_al-" "dynn_Dakhwar" }
+			{ "dynnp_al-" "dynn_Suri" }
+			"dynn_Khorasani"
+			{ "dynnp_al-" "dynn_Hasanid" }
+			"dynn_Aqabid"
+		}
+
+		dynasty_names = {
+			"dynn_Nasrid"
+			"dynn_Muayyadid"
+			"dynn_Isavid"
+			"dynn_Shukrid"
+			"dynn_Yaruqid"
+			"dynn_Qaimid"
+			"dynn_Alaid"
+			"dynn_Zahirid"
+			"dynn_Jamalid"
+			"dynn_Izzid"
+			"dynn_Bahaid"
+			"dynn_Shihabid"
+			"dynn_Jalalid"
+			"dynn_Yaminid"
+			"dynn_Lu_Lu"
+			"dynn_Sulaymanid"
+			"dynn_Mahmudid"
+			"dynn_Bahadur"
+			"dynn_Ata-Malik"
+			"dynn_Ma_anid"
+			"dynn_Hamdanid"
+			"dynn_Farooqid"
+			"dynn_Elbazid"
+			"dynn_Abbarid"
+			"dynn_Abbasid"
+			"dynn_Shamid"
+			"dynn_Allawid"
+			"dynn_Aqilid"
+			"dynn_Qahtanid"
+			"dynn_Bahjatid"
+			"dynn_Bekkalid"
+			"dynn_Baradid"
+			"dynn_Bseisoid"
+			"dynn_Fattaid"
+			"dynn_Fazilid"
+			"dynn_Gheisarid"
+			"dynn_Hayekid"
+			"dynn_Ibrahimid"
+			"dynn_Abdel_Nour"
+			"dynn_Abdilla"
+			"dynn_Mahdi"
+			{ "dynnp_el-" "dynn_Khattab" }
+			"dynn_Moneim"
+			"dynn_Moosevi"
+			"dynn_Mubarak"
+			"dynn_Moustapha"
+			"dynn_Nader"
+			"dynn_Nafti"
+			"dynn_Alam"
+			"dynn_Naeemi"
+			"dynn_Nahhas"
+			"dynn_Nimri"
+			"dynn_Himsi"
+			"dynn_Haleem"
+			"dynn_Qaderi"
+			"dynn_Aqabid"
+			"dynn_Jalalid"
+			"dynn_Jarraid"
+			"dynn_Arishid"
+			"dynn_Jasemid"
+			"dynn_Abdelid"
+			"dynn_Alivid"
+			"dynn_Qasimid"
+			"dynn_Shaddad"
+			{ "dynnp_al-" "dynn_Nafis" }
+			{ "dynnp_al-" "dynn_Baghdadi" }
+			{ "dynnp_al-" "dynn_Dakhwar" }
+			{ "dynnp_al-" "dynn_Suri" }
+			"dynn_Khorasani"
+			{ "dynnp_al-" "dynn_Hasanid" }
+			"dynn_Aqabid"
+		}
+
+		male_names = {
+			Abdul-Azeem Abdul-Hamid Abdul-Lateef Abdul-Razzaq Abdul-Wahad Abdullah Adhid Adil Akab Ala_i Ali Amir Ashraf Aziz Bashar Boutros Da_ud Faisal
+			Fayiz Ghazi Hakam Hamdan Hashmaddin Hussayn Ibrahim Imamaddin Isa Is_mail Ismat Jahan Jamil Kamran Khalid Khudayar Majid Mas_ud
+			Muaddam Muhammad Muhsin Mukhtar Murad Musa Muzaffar Muzaffaraddin Nasraddin Nuraddin Nuri Radhi Rasul Sa_ud Salahaddin Seyfullah Sharaf
+			Sirhan Taimur Tawfiq Umaya Wahid Yahya Ya_qub Yasar Youkhanna Zahir
+		}
+		female_names = {
+			Aisha Aziza Duha Farida Golbahar Gulnar Gulnaz Gulpari Hafsa Hasti Huma Ibtisam Jalila Kamala Karima Khandan Khaterah Khatira Kinah Lala Madiha Mahdokht
+			Mahsa Mahwash Marwa Mastana Mehri Mina Minoo Munawwar Muzhdah Nahal Najela Nasiba Nastaran Nazaneen Nazy Neelab Negar Niki Nikoo Noushin Rabiya
+			Rajiya Sadiya Salma Sitara Thana Zahra Zuhayra
+		}
+		dynasty_of_location_prefix = "dynnp_mn"
+		
+		patronym_prefix_male = "dynnpat_pre_ibn"
+		patronym_prefix_female = "dynnpat_pre_bint"
+		
+		dynasty_title_names = yes
+		founder_named_dynasties = yes
+
+		ethnicities = {
+			10 = arab
+		}
+
+		mercenary_names = {
+			{ name = "mercenary_company_sarkans_of_place" }
+			{ name = "mercenary_company_company_of_the_oasis" }
+			{ name = "mercenary_company_desert_brethren" }
+		}
+	}
+	egyptian = {
```

Comment:
The name is misleading in the mod, while the code name can stay like that for compatibility (although I would add a comment to say Egyptian Arabic directly here too) we should overwrite in localisation